packages/tspd/src/gen-extern-signatures/components/decorators-signatures.tsx

```diff
@@ -14,7 +14,7 @@ import { createTspdContext, TspdContext, useTspd } from "./tspd-context.js";
 export interface DecoratorSignaturesProps {
   decorators: DecoratorSignature[];
   namespaceName: string;
-  dollarDecoratorsRefKey: ay.Refkey;
+  dollarDecoratorsRefKey: Refkey;
```

Comment:
Need to import `type Refkey`, `For`, `refkey`, and `StatementList` -  and remove the `*` imports.